packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -43,30 +46,29 @@ export class EditorPage {
 	 * Constructs an instance of the component.
 	 *
 	 * @param {Page} page The underlying page.
+	 * @param param0 Keyed object parameter.
+	 * @param param0.target Site type. Defaults to 'simple'.
 	 */
-	constructor( page: Page ) {
+	constructor( page: Page, { target = 'simple' }: { target?: 'simple' | 'atomic' } = {} ) {
+		let editor: FrameLocator | Locator;
```

Comment:
If Atomic, since we are able to interact directly with the page, we set the `editor` to `Locator`.
If Simple, we continue to set the `editor` to `FrameLocator`.

Both `FrameLocator` and `Locator` to the page point to the top-level editor object for each respective editor type. Thus in this instance, callable methods for Atomic Editor and Simple Editor are both identical.

Reply:
Put another way, `editor` basically becomes a frame-safe way to locate the top div of the editor!

Reply:
Ahhh that's a good idea. I'll experiment with it a little bit!

Reply:
This was a great suggestion @dpasque, thanks! It makes for even cleaner method calls for the subcomponent classes of the editor.

One minor change, I've renamed the `wpAdminEditor` and `framedEditor` selectors to reflect the broadening of the definition.

Instead of `div[id="editor"]`, I've broadened the selector to be `body.block-editor-page`. 
This is for a couple of reasons:
- better simulation of the Frame/Page interaction prior to this refactor.
- visibility into transient elements (eg. dialog, modals, toasts) that are not necessarily within the `div[id="editor"]` element in the DOM.

The second reason is a bigger driver for the change in selector.

Reply:
This looks awesome! 👌